configure.ac

```diff
@@ -1671,10 +1671,10 @@ if test -z "$ssl_backends" -o "x$OPT_SSL" != xno &&
        ],
        [
          AC_MSG_RESULT(no)
-         dnl ok, so what about bouth -ldl and -lpthread?
+         dnl ok, so what about both -ldl and -lpthread?
 
          AC_MSG_CHECKING([OpenSSL linking with -ldl and -lpthread])
-         LIBS="$CLEANLIBS -ldl -lpthread -lcrypto"
+         LIBS="$CLEANLIBS -lcrypto -ldl -lpthread"
```

Comment:
I'm not entirely sure why this suddenly uses CLEANLIBS, where the one above uses LIBS.

Reply:
I think maybe the previous one should use CLEANLIBS too, as LIBS already contains "-lcrypto" at that point and it adds it again...

Reply:
Yes. I did wonder about that. Worth trying do you think?

Reply:
It's not the end of the world, but sure, if you fixup that at the same time as this fix, I approve!

Reply:
Done.